contracts/ManagerProxy.sol

```diff
@@ -4,48 +4,42 @@ import "./ManagerProxyTarget.sol";
 
 
 contract ManagerProxy is ManagerProxyTarget {
-    function ManagerProxy(address _controller, bytes32 _targetContractId) Manager(_controller) {
+    function ManagerProxy(address _controller, bytes32 _targetContractId) public Manager(_controller) {
```

Comment:
This means Manager can be called by anyone right?

Reply:
Yeah, but its the constructor so it is only ever called once during contract creation. The constructor is `public` by default anyway - the explicit `public` is added to get rid of the compiler warning